run.sh

```diff
@@ -240,6 +240,7 @@ chmod +x /etc/service/${DAEMON}/run
 if [ "${NETWORK}" == "regtest" ]; then
     if [ "${DAEMON}" != "liquid" ]; then
         /srv/explorer/bitcoin/bin/bitcoind -conf=/data/.bitcoin.conf -datadir=/data/bitcoin -daemon -regtest
+        cli -rpcwait createwallet "default"
```

Comment:
This would fail when the wallet already exists. Maybe do something like `cli -rpcwait loadwallet default ||  cli createwallet default`?